src/components/data-management/MetadataEditor.jsx

```diff
@@ -1,26 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import {
-  Popover, Button, Space, Divider,
+  Popover, Button, Space, Divider, Tooltip,
 } from 'antd';
 import { FormatPainterOutlined } from '@ant-design/icons';
 
 const MetadataEditor = (props) => {
   const {
-    onCreate, onCancel, children, massEdit, ...restOfProps
+    onReplaceEmpty,
+    onReplaceAll,
+    onClearAll,
+    massEdit,
+    children,
+    ...restOfProps
   } = props;
 
+  const [value, setValue] = useState('');
+
+  const onChange = (e) => setValue(e?.target?.value || e);
```

Comment:
To handle inputs whose return value is the value itself (`value`) or event object `e` (accessed via `e.target.value`)